catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -219,101 +330,48 @@ private Metacard transformPdf(String id, PDDocument pdfDocument, String contentI
      * @param metacard    A mutable metacard to add the extracted data to
      */
     private void extractPdfMetadata(PDDocument pdfDocument, MetacardImpl metacard) {
-        StringBuilder metadataField = new StringBuilder("<pdf>");
-
-        PDMetadata documentCatalogMetadata = pdfDocument.getDocumentCatalog()
-                .getMetadata();
-        if (documentCatalogMetadata != null) {
-            try (InputStream inputStream = documentCatalogMetadata.createInputStream()) {
-                metadataField.append(IOUtils.toString(inputStream));
-            } catch (IOException e) {
-                LOGGER.warn("Couldn't read the PDF document catalog's metadata", e);
-            }
-        }
 
         PDDocumentInformation documentInformation = pdfDocument.getDocumentInformation();
 
         Calendar creationDate = documentInformation.getCreationDate();
         if (creationDate != null) {
             metacard.setCreatedDate(creationDate.getTime());
-            addXmlElement("creationDate", DATE_FORMAT.format(creationDate), metadataField);
             LOGGER.debug("PDF Creation date was: {}", DATE_FORMAT.format(creationDate));
         }
 
         Calendar modificationDate = documentInformation.getModificationDate();
         if (modificationDate != null) {
             metacard.setModifiedDate(modificationDate.getTime());
-            addXmlElement("modificationDate", DATE_FORMAT.format(modificationDate), metadataField);
             LOGGER.debug("PDF Modification date was: {}", DATE_FORMAT.format(modificationDate));
         }
 
-        String title = documentInformation.getTitle();
-        if (StringUtils.isNotBlank(title)) {
-            metacard.setTitle(title);
-            addXmlElement("title", title, metadataField);
+        if (usePdfTitleAsTitle) {
+            String title = documentInformation.getTitle();
```

Comment:
It might be worth extracting the "get value -> check not blank -> set attribute" pattern into a private method since you use it in multiple places.